tooling/bench/tests/cpu_intensive/public/site.js

```diff
@@ -17,13 +17,7 @@ const onMessage = (message) => {
 
   if (message.data.status === 'done') {
     // tell tauri that we are done
-    window.__TAURI__.invoke('tauri', {
-      __tauriModule: 'Process',
-      message: {
-        cmd: 'exit',
-        exitCode: 0
-      }
-    })
+    window.__TAURI__.invoke('app_completed_successfully');
```

Comment:
why not use `window.__TAURI__.process.exit`?

Reply:
Let see if I got the same error than before

Reply:
it die with `pure virtual method called` (exit code 1)

Reply:
on `hello world only`, i guess it's because the init sequence is not fully done and it cause a stack error

Reply:
but with the command it seems to work fine (on the CI and locally)
https://github.com/tauri-apps/tauri/runs/3281557580?check_suite_focus=true#step:11:524

Reply:
hmm weird